modules/app.rb

```diff
@@ -0,0 +1,166 @@
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'person'
+
+class App
+  attr_accessor :books, :rentals, :people
+
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def selected_option(option)
+    case option
+
+    when '1'
+      list_books
+    when '2'
+      list_people
```

Comment:
[OPTIONAL]  For a better user experience, Kindly add some friendly text to display to a user when there's no book or person on the lists😉 Same when the user exits the app😉 you can add a friendly text appreciating the user for their usage before the app is terminated😉